libraries/azure-app-configuration-importer/src/internal/parsers/defaultConfigurationSettingsConverter.ts

```diff
@@ -193,14 +281,15 @@ class FeatureFlagConfigurationSettingsConverter implements ConfigurationSettings
     }
   }
 
-  private getFeatureFlagValue(
+  private getFeatureFlagDefinitionFromLegacySchema(
```

Comment:
This function returns a feature flag value ? I would suggest renaming it to `getFeatureFlagValue`

Reply:
According to Zhenlan's suggestion, let's rename it to `getFeatureFlagValueFromDotnetSchema`